docs/source/trading-calendars.rst

```diff
@@ -0,0 +1,172 @@
+
+In order for you to backtest your strategy, the dates in that are accounted for in your `data bundle <http://www.zipline.io/bundles.html>`__ and the dates in your ``TradingCalendar`` should match up; if the dates don't match up, then you you're going to see some errors along the way. This holds for both minutely and daily data.
+
+
+The TradingCalendar Class
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``TradingCalendar`` class has many properties we should be thinking about if we were to build our own ``TradingCalendar`` for an exchange. These include properties such as:
+
+  - Name of the Exchange
+  - Timezone
+  - Open Time
+  - Close Time
+  - Regular & Adhoc Holidays
+  - Special Opens & Closes
+
+And several others. If you'd like to see all of the properties and methods available to you through the ``TradingCalendar`` API, please take a look at the `API Reference <http://www.zipline.io/appendix.html#trading-calendar-api>`__
+
+Now we'll take a look at the London Stock Exchange Calendar :class:`~zipline.utils.calendars.exchange_calendar_lse.LSEExchangeCalendar` as an example below:
+
+.. code-block:: python
+
+  class LSEExchangeCalendar(TradingCalendar):
+    """
+    Exchange calendar for the London Stock Exchange
+
+    Open Time: 8:00 AM, GMT
+    Close Time: 4:30 PM, GMT
+
+    Regularly-Observed Holidays:
+      - New Years Day (observed on first business day on/after)
+      - Good Friday
+      - Easter Monday
+      - Early May Bank Holiday (first Monday in May)
+      - Spring Bank Holiday (last Monday in May)
+      - Summer Bank Holiday (last Monday in May)
+      - Christmas Day
+      - Dec. 27th (if Christmas is on a weekend)
+      - Boxing Day
+      - Dec. 28th (if Boxing Day is on a weekend)
+    """
+
+    @property
+    def name(self):
+      return "LSE"
+
+    @property
+    def tz(self):
+      return timezone('Europe/London')
+
+    @property
+    def open_time(self):
+      return time(8, 1)
+
+    @property
+    def close_time(self):
+      return time(16, 30)
+
+    @property
+    def regular_holidays(self):
+      return HolidayCalendar([
+        LSENewYearsDay,
+        GoodFriday,
+        EasterMonday,
+        MayBank,
+        SpringBank,
+        SummerBank,
+        Christmas,
+        WeekendChristmas,
+        BoxingDay,
+        WeekendBoxingDay
+      ])
+
+You can create the ``Holiday`` objects mentioned in ``def regular_holidays(self)` through the `pandas <http://pandas.pydata.org/pandas-docs/stable/>`__ module, ``pandas.tseries.holiday.Holiday``, and also take a look at the `LSEExchangeCalendar <https://github.com/quantopian/zipline/blob/master/zipline/utils/calendars/exchange_calendar_lse.py>`__ code as an example.
+
+
+Building a Custom Trading Calendar
```

Comment:
This section looks great - I'd just add a bit about how to create custom holidays on the trading calendar, for completeness